audio_test.go

```diff
@@ -19,11 +19,12 @@ func TestAudioWithFailingFormBuilder(t *testing.T) {
 	test.CreateTestFile(t, path)
 
 	req := AudioRequest{
-		FilePath:    path,
-		Prompt:      "test",
-		Temperature: 0.5,
-		Language:    "en",
-		Format:      AudioResponseFormatSRT,
+		FilePath:               path,
+		Prompt:                 "test",
+		Temperature:            0.5,
+		Language:               "en",
+		Format:                 AudioResponseFormatVerboseJSON,
+		TimeStampGranularities: []TimestampGranularity{TimestampGranularityWord},
```

Comment:
Thank you for the PR! Could we please keep this as a separate test?

Reply:
added a individual test `TestAudioTimeGranularities`